app/models/patient.rb

```diff
@@ -90,6 +91,11 @@ class Patient < ApplicationRecord
             absence: { message: "cannot be 'true' when 'Last Date of Exposure' is specified" },
             if: -> { last_date_of_exposure.present? }
 
+  validates :follow_up_note,
+            on: %i[api import],
+            absence: { message: "must be blank when '#{VALIDATION[:follow_up_reason][:label]}' is blank" },
+            if: -> { follow_up_reason.nil? }
```

Comment:
For awareness, I left this https://github.com/SaraAlert/SaraAlert/pull/983#discussion_r669066831, but I would expect that to be fixed there rather than here